frontend/src/DiscoverMovie/Overview/AddListMovieOverview.js

```diff
@@ -138,6 +143,26 @@ class AddListMovieOverview extends Component {
                 {...linkProps}
               >
                 {title}({year})
+
+                <span style={{ marginLeft: `${linkMenuMargin}em` }} className={styles.links}>
```

Comment:
since linkMenuMargin is a constant value, just added it to the `AddListMovieOverview.css` file for styles.links